File: Java/abaixo_da_media.java
import java.util.Locale;
import java.util.Scanner;

public class abaixo_da_media {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int n;
		double soma, media;

		System.out.print("Quantos elementos vai ter o vetor? ");
		n = sc.nextInt();

		double[] vet = new double[n];

		for (int i = 0; i < n; i++) {
			System.out.print("Digite um numero: ");
			vet[i] = sc.nextDouble();
		}

		soma = 0.0;
		media = 0.0;
		for (int i = 0; i < n; i++) {
			soma = soma + vet[i];
		}

		media = soma / n;
		System.out.println();
		System.out.printf("MEDIA DO VETOR = %.3f\n", media);

		System.out.println("ELEMENTOS ABAIXO DA MEDIA:");
		for (int i = 0; i < n; i++) {
			if (vet[i] < media) {
				System.out.printf("%.1f\n", vet[i]);
			}
		}

		sc.close();
	}
}

File: Python/abaixo_da_media.py
soma: float; media: float
n = int(input("Quantos elementos vai ter o vetor? "))
vet = [0 for x in range(n)]

for i in range(0, n):
    vet[i] = float(input("Digite um numero: "))

soma = 0;
for i in range(0, n):
    soma = soma + vet[i]

media = soma / n
print(f"\nMEDIA DO VETOR = {media:.3f}")
print("ELEMENTOS ABAIXO DA MEDIA:")
for i in range(0, n):
    if vet[i] < media:
        print(f"{vet[i]:.1f}")

